src/ucp/core/ucp_worker.inl

```diff
@@ -194,7 +194,7 @@ ucp_worker_is_tl_2sockaddr(ucp_worker_h worker, ucp_rsc_index_t rsc_index)
 static UCS_F_ALWAYS_INLINE unsigned
 ucp_worker_common_address_pack_flags(ucp_worker_h worker)
 {
-    unsigned pack_flags = 0;
+    unsigned pack_flags = UCP_ADDRESS_PACK_FLAG_WORKER_UUID;
```

Comment:
do not pack uuid in private_data if not asked for: add endpoint create flag to pass uuid to server side